README.md

```diff
@@ -38,12 +38,13 @@ Last updated: _2024-05-10_
 
 These are the MLflow objects and their attributes that can be exported.
```

Comment:
are there any objects that are known to be unsupported for export? add a row for them and indicate something like "currently unsupported"